operators/src/source/ogr_source.rs

```diff
@@ -1058,14 +1058,13 @@ mod tests {
                 "layerName": "foobar",
                 "dataType": "MultiPoint",
                 "time": {
-                    "start": {
-                        "startField": "start",
-                        "startFormat": {
-                            "format": "custom",
-                            "customFormat": "YYYY-MM-DD"
-                        },
-                        "duration": 42
-                    }
+                    "type": "start",
+                    "startField": "start",
+                    "startFormat": {
+                        "format": "custom",
+                        "customFormat": "YYYY-MM-DD"
+                    },
+                    "duration": 42
```

Comment:
I'm not sure if this improves this setting since we could specify "start" twice now

Reply:
please elaborate, I don't get the problem

Reply:
We had fields in the dict, namely start, and end.
And now you add basically just a list of dicts and specify it via *type*.
This doesn't pose an improvement I guess, but on the other hand a user can specify "start" twice.